spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -1811,6 +1811,26 @@ public static class Admin {
 		 */
 		private Set<String> tlsProtocols;
 
+		/**
+		 * Connection time out.
+		 */
+		private Duration connectTimeout = Duration.ofMinutes(1);
```

Comment:
This will work as expected once https://github.com/apache/pulsar/pull/17375 gets merged. We ran into this originally in https://github.com/apache/pulsar/issues/17333